app/javascript/markdown-it-container-speak.js

```diff
@@ -0,0 +1,26 @@
+import MarkdownItContainer from 'markdown-it-container'
+
+export default (md) => {
+  md.use(MarkdownItContainer, 'speak', {
+    render: (tokens, idx) => {
+      const speakerName = escapeHTML(tokens[idx].info)
+        .replace('speak @', '')
+        .trim()
+
+      if (tokens[idx].nesting === 1) {
+        return `<div class="speak"><div class="speak__speaker"><a href="/users/${speakerName}" class="a-user-emoji-link"><img class="js-user-icon a-user-emoji" data-user="${speakerName}" width="40" height="40"></a></div><div class="speak__body">`
```

Comment:
HTMLのタグを複数行で記述すると、ボディ部分の構成がより分かりやすくなるかもと思いました。

```suggestion
        return `<div class="speak"><div class="speak__speaker">
                  <a href="/users/${speakerName}" class="a-user-emoji-link">
                    <img class="js-user-icon a-user-emoji" data-user="${speakerName}" width="40" height="40">
                  </a>
                </div>
                <div class="speak__body">`
```